src/Roslyn.Diagnostics.Analyzers/Setup/source.extension.vsixmanifest

```diff
@@ -7,7 +7,7 @@
     <Description xml:space="preserve">Roslyn.Diagnostics Analyzers</Description>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0.0.0,17.0.0.0)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0.27130.0,]" />
```

Comment:
This should not be reverted

Reply:
Or will this be restored in 3.0.x branch? I am fine with that as well...

Reply:
This has to be reverted, or the 2.9 builds will stop being installable in VS2017.

Reply:
We can address installation in VS2019 as a follow-up. There are multiple options.

Reply:
Ah got you. That makes sense.